sktime/forecasting/trend/_spline_trend_forecaster.py

```diff
@@ -0,0 +1,215 @@
+# !/usr/bin/env python3 -u
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements SplineTrendForecaster."""
+
+__author__ = ["tensorflow-as-tf", "mloning", "aiwalter", "fkiraly", "jgyasu"]
```

Comment:
I think this may be just you, if this class is yours entirely. If you made this way to note that it was inspired by polynomial trend one, I believe you can consider inheriting or similar if there is a significant amount of code overlap.

(Just my opinoin, not necessarily of all core developers)